app/src/app/shared/components/information/information.component.ts

```diff
@@ -23,12 +23,38 @@ export class InformationComponent {
   @Input()
   sourceID: SourceID[];
 
+  sourceIDObject: any;
+
   constructor() {}
 
-  ngOnChanges() {
+  ngOnChanges(changes: SimpleChanges) {
+    if(changes.sourceID) {
+      this.handleSourceIDArray(changes.sourceID.currentValue);
+    }
     this.checkRequiredFields();
   }
 
+  /**
+   * This Method converts sourceID Array to an object and checks
+   * if the element is a getty- or dnb-element and 'renames'
+   * the key to getty or dnb. These keys are checked inside
+   * the html-template
+   */
+  handleSourceIDArray(sID: SourceID[]) {
```

Comment:
I think we should collect all possible values for source together with potential labels. 
These are all possibilities with their Names:

```
'http://vocab.getty.edu/ulan' : 'Union List of Artist Names ID',
'http://vocab.getty.edu/tgn' : 'Getty Thesaurus of Geographic Names ID',
'http://vocab.getty.edu/aat' : 'Art & Architecture Thesaurus ID',
'ISIL (ISO 15511)' : 'International Standard Identifier for Libraries and Related Organizations (ISIL)',
'http://d-nb.info/gnd' : 'German National Library ID'
```

It would be nice if there is a possibility to access the labels simply by `sourceId.label` without any other pre-assignment, but i think thats not that easy/possible. At least we could pre-assign enum values to a new SourceID util-attribute (eg label):

utils.ts or somewhere:
```
export interface SourceID {
  source: string;
  id: string;
  term?: string;
  label?: string;
}

[...]

export enum SourceToLabel {
  'http://vocab.getty.edu/ulan' = 'Union List of Artist Names ID',
  'http://vocab.getty.edu/tgn' = 'Getty Thesaurus of Geographic Names ID',
  'http://vocab.getty.edu/aat' = 'Art & Architecture Thesaurus ID',
  'ISIL (ISO 15511)' = 'International Standard Identifier for Libraries and Related Organizations (ISIL)',
  'http://d-nb.info/gnd' = 'German National Library ID'
}
```

in information.component.ts:
```
    sIDs.map(sID => sID.label = SourceToLabel[sID.source]); //probably check if sID.source is in enum keys
```
html:
```
      <a *ngFor="let sID of sourceID" href="{{ sID.id }}" target="_blank">{{ sID.label }}</a>
```